pokemon-app/src/components/EnemyPokemon.js

```diff
@@ -1,41 +1,94 @@
-import React from "react";
+import React, { useState } from "react";
 import fightPad from "../assets/fightpad.png";
 import "../styles/EnemyPokemon.css";
+import ThemeSongs from "./ThemeSongs";
+import fightSong from "../assets/fightSong.mp3"
+import Types from "./Types";
 
-const EnemyPokemon = ({ enemyPokemon }) =>
-  !enemyPokemon ? null : (
+const EnemyPokemon = ({ enemyPokemon, hasEnemy }) => {
+  const [enemyStatsOnTop, setEnemyStatsOnTop] = useState(true);
+
+  return (
     <>
-      <div className="enemy-pokemon">
-        <div className="enemy-pokemon__display">
-          <div className="enemy-pokemon__display-name">
-            <p className="paragraph">{enemyPokemon?.name}</p>
+      {!enemyPokemon ? null : (
+        <>
+          <div className="enemy-pokemon">
+            <div className="enemy-pokemon__display">
+              <div className="enemy-pokemon__display-name">
+                <p className="paragraph">{enemyPokemon?.name}</p>
+              </div>
+
+              <div className="enemy-pokemon__display-image">
+                <img
+                  src={`${enemyPokemon?.sprites?.front_default}`}
+                  alt={`Front default of ${enemyPokemon?.name} sprite`}
+                />
+              </div>
+            </div>
+
+            <div className="enemy-pokemon-types">
+              <Types pokemon={enemyPokemon} />
+            </div>
+
+            <div
+              onClick={() => setEnemyStatsOnTop(true)}
+              className={`enemy-pokemon__stats-title ${
+                enemyStatsOnTop ? "on-tip-top" : ""
+              }`}
+            >
```

Comment:
Stuff like this makes a great use case for styled components.  With styled components, you can adjust the styling based on props.